generate.py

```diff
@@ -210,7 +210,7 @@ def disable(self, excluded):
                     module.modified = True
 
     def suite_count(self):
-        return len(self.modules)
+        return sum(len(module.initializers) for module in self.modules.values())
```

Comment:
This needs to be `sum(max(1, len(m.initializers)) for m in self.modules.values())`. Otherwise it won't account for modules without initializers at all.